_data/simple-icons.json

```diff
@@ -12082,6 +12082,11 @@
             "hex": "3F4F75",
             "source": "https://plotly.com"
         },
+        {
+            "title": "Plume",
+            "hex": "7C5CDF",
+            "source": "https://www.plume.com"
```

Comment:
https://www.plume.com/legal/trademarks/ should be linked here. I can never remember if we can include icons with these kind of terms - since they do not allow certain use cases that our users may or may not do are we required to seek permission @adamrusted?

Reply:
https://github.com/simple-icons/simple-icons/issues/9574#issuecomment-2149527091 said it's fine, but we can still email them for approval if needed

Reply:
Right I forgot about my own comment 😅 ignore me!